airflow/models/taskinstance.py

```diff
@@ -187,19 +187,28 @@ def set_current_context(context: Context) -> Generator[Context, None, None]:
             )
 
 
-def stop_all_tasks_in_dag(tis: list[TaskInstance], session: Session, task_id_to_ignore: int):
+def _stop_remaining_tasks(*, dag: DAG, tis: list[TaskInstance], task_id_to_ignore: str, session: Session):
```

Comment:
What do you think about making this a little less generic, and instead of accepting a list of tis and a single "task_id_to_ignore", let it use self to query and ignore all right here?

Reply:
no problem